content/post/2020-05-29-distribute-data/index.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+slug: distribute-data
+title: "How to distribute data with your R packages"
+authors:
+  - Maëlle Salmon
+date: "2020-05-29"
+tags:
+- package development
+- standards
+- data
+output: hugodown::hugo_document
+---
+
+```{r setup, include=FALSE}
+library("printr")
+# Options to have images saved in the post folder
+# And to disable symbols before output
+knitr::opts_chunk$set(fig.path = "", comment = "")
+
+# knitr hook to make images output use Hugo options
+knitr::knit_hooks$set(
+  plot = function(x, options) {
+    hugoopts <- options$hugoopts
+    paste0(
+      "{{<figure src=",
+      '"', x, '" ',
+      if (!is.null(hugoopts)) {
+        glue::glue_collapse(
+          glue::glue('{names(hugoopts)}="{hugoopts}"'),
+          sep = " "
+        )
+      },
+      ">}}\n"
+    )
+  }
+)
+
+# knitr hook to use Hugo highlighting options
+knitr::knit_hooks$set(
+  source = function(x, options) {
+  hlopts <- options$hlopts
+    paste0(
+      "```r ",
+      if (!is.null(hlopts)) {
+      paste0("{",
+        glue::glue_collapse(
+          glue::glue('{names(hlopts)}={hlopts}'),
+          sep = ","
+        ), "}"
+        )
+      },
+      "\n", glue::glue_collapse(x, sep = "\n"), "\n```\n"
+    )
+  }
+)
+```
+
+Distributing data with an R package can be crucial for the package or even the only goal of a package:
+to show what a function can accomplish with a dataset; to show how a package can help tidy a messy data format; to test the package; for teach purposes; to allow users to directly use instead of fetching and cleaning the data.
+Now, _how_ to provide data with/for your package is a recurring theme in [R package development channels](/2019/04/11/r-package-devel/).
+In this post, we shall present various ways to distribute data with/for an R package, depending on the data use case and on its size.
+
+_Thanks to the R connoisseurs [Thomas Vroylandt](https://tvroylandt.netlify.app/), [Sébastien Rochette](https://statnmap.com/) and [Hugo Gruson](https://www.normalesup.org/~hgruson/) for providing some inspiration and resources for this post! :pray: [^grrr]_
+
+## Data in your package
+
+Sometimes the data can be *vendored*[^vendor] i.e. live in your package source, and even in your package built and installed versions.
```

Comment:
even in your package built and installed versions  -> and even built into and installed with the package?